README.md

```diff
@@ -1,5 +1,8 @@
 # graft
 
+![Gem Version](https://img.shields.io/gem/v/graft?style=for-the-badge&logo=rubygems)
+![GitHub Actions Workflow Status](https://img.shields.io/github/actions/workflow/status/Datadog/graft-rb/test.yml?style=for-the-badge&logo=github)
```

Comment:
I really don't find those useful, and in fact misleading:

- last version is on the right side (or it should be once we have release pages in GitHub, which we should), plus it doesn't reflect the actual commit shown
- test result is not the one for the current commit

Reply:
> last version is on the right side (or it should be once we have release pages in GitHub, which we should), plus it doesn't reflect the actual commit shown

It shown the last version released on Rubygems.org. I don't mind if we have release pages in GitHub.

>test result is not the one for the current commit
I can add branch as parameters.